Ryujinx.Ui.Common/App/ApplicationLibrary.cs

```diff
@@ -112,9 +112,9 @@ public void LoadApplications(List<string> appDirs, Language desiredTitleLanguage
                             {
                                 return;
                             }
-                        
+
                             string extension = Path.GetExtension(app).ToLower();
-                        
+
                             if (!File.GetAttributes(app).HasFlag(FileAttributes.Hidden) && extension is ".nsp" or ".pfs0" or ".xci" or ".nca" or ".nro" or ".nso")
                             {
                                 applications.Add(app);
```

Comment:
```suggestion
                            if (!fileInfo.Attributes.HasFlag(FileAttributes.Hidden) && extension is ".nsp" or ".pfs0" or ".xci" or ".nca" or ".nro" or ".nso")
                            {
                                var fullPath = fileInfo.ResolveLinkTarget(true)?.FullName ?? fileInfo.FullName;

                                applications.Add(fullPath);
```

Now we only need to add a check for symlinks here and resolve the actual path before we add it to the list of applications.